### PR TITLE
Updating pwd in msdeploy doc to get the value from an env var

### PR DESCRIPTION
**Pull request by sayedihashimi**
In a previous version of this doc, the fictitious password was in plain text. I've modified the sample, and doc, to pull the pwd value from an environment variable, instead of hard coding it.

cc @Rick-Anderson @vijayrkn

### CODE REVIEW

File: aspnet/web-forms/overview/deployment/configuring-server-environments-for-web-deployment/choosing-the-right-approach-to-web-deployment.md [hunk 1]
```diff
@@ -91,7 +91,8 @@ In the case of non-administrator users, the Web Management Service (WMSvc) will
...
+For example, suppose a build process is configured to automatically deploy a web application to a staging environment after every successful build. If you used the remote agent approach, you'd need to make the build process identity an administrator on your destination servers. In contrast, using the Web Deploy Handler approach you can give a non-administrator user&#x2014;**FABRIKAM\stagingdeployer** in this case&#x2014;permission to a specific IIS website only, and the build process can provide these credentials to deploy the web package. Note the following example is using `%ContactManagerPublishPassword%`, which is pulling the password value from an environment variable. To successfuly execute the script, that environment variable will need to be defined with
+the correct value.
```
Comment (Rick-Anderson) [inline aspnet/web-forms/overview/deployment/configuring-server-environments-for-web-deployment/choosing-the-right-approach-to-web-deployment.md:95]:
```suggestion
```suggestion
For example, suppose a build process is configured to automatically deploy a web application to a staging environment after every successful build. If you used the remote agent approach, you'd need to make the build process identity an administrator on your destination servers. In contrast, using the Web Deploy Handler approach you can give a non-administrator user&#x2014;**FABRIKAM\stagingdeployer** in this case&#x2014;permission to a specific IIS website only, and the build process can provide these credentials to deploy the web package. Note the following example is using `%ContactManagerPublishPassword%`, which is pulling the password value from an environment variable. To successfully execute the script, `%ContactManagerPublishPassword%`  variable must be defined with the correct value.
```
```

**Review by Rick-Anderson — APPROVED** [review]
Fantastic. Much better. Much appreciated.

File: aspnet/web-forms/overview/deployment/configuring-server-environments-for-web-deployment/choosing-the-right-approach-to-web-deployment.md (hunk 1 above)
Comment (sayedihashimi) [reply aspnet/web-forms/overview/deployment/configuring-server-environments-for-web-deployment/choosing-the-right-approach-to-web-deployment.md:95]: Thanks @Rick-Anderson, I'd like to merge this suggestion but I see there is a ` ```suggestion ` on line 94, is that correct? Should I commit your suggestion in the github interface, or should I take your changes and commit them directly?

Comment (Rick-Anderson) [reply aspnet/web-forms/overview/deployment/configuring-server-environments-for-web-deployment/choosing-the-right-approach-to-web-deployment.md:95]: Yes, on the [files tab](https://github.com/dotnet/AspNetDocs/pull/523/files), select commit. If there were multiple, better to use **Add suggestion to batch**. Then if you need to do more work on your local computer, do a `git pull origin myBranch`. In this case you only need to commit.

**Pull request merged**